### PR TITLE
[WIP]: make the data write amount normal distribution

### PR DESCRIPTION
**Pull request by Notdeath**
make the data write amount normal distribution, there are still some details that need to be improved.

### CODE REVIEW

File: pkg/client/client.go [hunk 1]
```diff
@@ -64,16 +65,16 @@ func newWorker(p *properties.Properties, threadID int, threadCount int, workload
...
-		fmt.Printf("totalOpCount(%s/%s/%s): %d should be bigger than threadCount: %d",
-			prop.OperationCount,
-			prop.InsertCount,
-			prop.RecordCount,
-			totalOpCount,
-			threadCount)
-
-		os.Exit(-1)
-	}
+	//if totalOpCount < int64(threadCount) {
```
Comment (lhy1024) [inline pkg/client/client.go]: I think it is not appreciated to use comments here.

File: pkg/client/client.go [hunk 2]
```diff
@@ -114,22 +115,31 @@ func (w *worker) run(ctx context.Context) {
...
+	u := w.p.GetFloat64(prop.ExpectedValue,prop.ExpectedValueDefault)
+	a := w.p.GetFloat64(prop.StandardDeviation,prop.StandardDeviationDefault)
+	t := w.p.GetFloat64(prop.TimeDelay,prop.TimeDelayDefault)
 
 	for w.opCount == 0 || w.opsDone < w.opCount {
 		var err error
 		opsCount := 1
 		if w.doTransactions {
 			if w.doBatch {
+				//fmt.Println("doBatchTransaction\n")
```
Comment (lhy1024) [inline pkg/client/client.go]: would you like to replace debug info with `log.debug`?

File: pkg/client/client.go [hunk 3]
```diff
@@ -114,22 +115,31 @@ func (w *worker) run(ctx context.Context) {
 	}
 
 	startTime := time.Now()
+	u := w.p.GetFloat64(prop.ExpectedValue,prop.ExpectedValueDefault)
```
Comment (lhy1024) [inline pkg/client/client.go]: we can use more meaningful name.

**Comment by disksing** [comment]
closing due to no respond.

**Pull request closed without merging**